src/client/index.css

```diff
@@ -3,7 +3,7 @@
 
 * {
   box-sizing: border-box;
-  color: #3b4349;
+  color: var(--dark-blue);
```

Comment:
`background: var(--font-color);`

Reply:
Oh sorry this file is actually index.css 🙈 I think you shouldn't edit this file in your PR. 
Maybe that's what Niels means on his comment. Your PR should only have the changes about your task. Otherwise it will be really complicated to solve merge conflicts with other PRs and develop branch.